### PR TITLE
Splitting up compat data for break-after

### PR DESCRIPTION
**Pull request by rachelandrew**
To match other updated compat data. I am splitting up the data into multicol, paged media and regions i order to better show support in these different contexts.

Related to MDN issue: https://github.com/mdn/sprints/issues/734

Also compat data PRs:

* https://github.com/mdn/browser-compat-data/pull/3327
* https://github.com/mdn/browser-compat-data/pull/3315

### CODE REVIEW

File: css/properties/break-after.json [hunk 1]
```diff
@@ -82,93 +28,365 @@
...
               "deprecated": false
             }
+          },
+          "column": {
+            "__compat": {
+              "description": "<code>column</code> and <code>avoid-column</code>",
+              "support": {
+                "chrome": {
+                  "version_added": "50",
+                  "notes": "Supports <code>column</code> but not <code>avoid-column</code>."
```
Comment (ddbeck) [inline css/properties/break-after.json]: Thanks for this @rachelandrew! Just one area I had questions about.

* If this features includes `column` and `avoid-column`, then I think this entry (and the IE entry below) needs to have `"partial_support": true` or we need to consider splitting up `column` and `avoid-column` into separate features
* If Chrome only supports `column` then I'm skeptical that Chrome for Android and Webview have support for `avoid-column`. Is that something we can substantiate? Or maybe it doesn't matter if the feature is split in two

Comment (rachelandrew) [reply css/properties/break-after.json]: @ddbeck you are right, so I guess we could split them or just put a note as I have for the other one. I am fine with that either way, do you have a preference?

Comment (ddbeck) [reply css/properties/break-after.json]: I guess I prefer splitting them in two. Sorry for the delay on answering your question, @rachelandrew.

Comment (rachelandrew) [reply css/properties/break-after.json]: @ddbeck I've just pushed a commit splitting these.

Comment (ddbeck) [reply css/properties/break-after.json]: Thank you!

**Pull request merged**